src/collection.cr

```diff
@@ -266,6 +293,7 @@ end
 
 struct HTTP::Params
```

Comment:
This one's [moved to `URI::Params`](https://github.com/crystal-lang/crystal/pull/10098).